pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ASTFormalComment.java

```diff
@@ -28,15 +28,8 @@ public String getImage() {
         return token.getText();
     }
 
-    /* TODO: needed?
-    public Chars getToken() {
-        String text = token.getText();
-        return Chars.fromString(text, 0, text.length());
-    }
```

Comment:
original implementation (not sure if you restored it, I'm not through all the commits yet):

```
    public Chars getToken() {
        return image;
    }
```

Using Chars instead of String (getImage) might help a bit for performance/memory usage, as no String instance needs to be created (Chars is a CharSequence as well) in ApexDocRule.

Reply:
Ok, that's not relevant anymore.